RFC.md

```diff
@@ -1,9 +1,18 @@
 # RFC: Amplify JS TypeScript Improvements
+Amplify JS is looking to improve our TypeScript support across the library to better serve our customers and provide a more intuitive & idiomatic developer experience. To this end we are requesting community feedback on a variety of changes and improvements that will be available to customers in the near future.
```

Comment:
To this end, we are requesting your feedback.

That will be available to you in our next major version release